preprocess_text.py

```diff
@@ -0,0 +1,78 @@
+import nltk
+import gensim
+from nltk.tokenize import word_tokenize
+
+
+def tokenize_a_doc(s):
+    try:
+        tok = word_tokenize(' '.join(s.split(" ")).lower())
+    except:
+        tok = ""
+    return tok
+
+def vectorize_text(L):
```

Comment:
I think this can probably be a list-comprehension
[ vect.append(wdic.get(word)) for word in l if wdic.get(word) is not None else vect.append(0)]